Makefile

```diff
@@ -47,47 +47,52 @@ help:
 
 .PHONY: clean
 clean:
-	rm -rf $(BUILDDIR)/*
+	rm -rf $(BUILDDIR)/* contracts
+
+# download `raiden-contracts/raiden_contracts/contracts` to the local `contracts` dir
+# This is skipped if the dir already exists. Use `make clean` to force an update.
+contracts:
+	curl -sSL https://github.com/raiden-network/raiden-contracts/tarball/master | tar xvz --wildcards '*/raiden_contracts/contracts' --strip-components 2
```

Comment:
I get the following error on macOS, can we workaround that?
```sh
curl -sSL https://github.com/raiden-network/raiden-contracts/tarball/master | tar xvz --wildcards '*/raiden_contracts/contracts' --strip-components 2
tar: Option --wildcards is not supported
```

Reply:
Additionally, is this run on readthedocs?

Reply:
Apparently `--wildcards` is a gnu extension and the BSD tar is missing that. I'll find an alternative way of doing that.

Why should `readthedocs` be a problem? I thought it was just a linux system with sphinx installed that runs the different `make` targets.

Reply:
Please try again on MacOS, now.